CONTRIBUTING.md

```diff
@@ -13,6 +13,23 @@ for a list of Miri maintainers.
 
 [Rust Zulip]: https://rust-lang.zulipchat.com
 
+### Pull review process
+
+When you get a review, please take care of the requested changes in new commits. Do not amend
+existing commits. Generally avoid force-pushing. The only time you should force push is when there
+is a conflict with the master branch (in that case you should rebase across master, not merge), and
+all the way at the end of the review process when the reviewer tells you that the PR is done and you
+should squash the commits. For the latter case, use `git rebase --keep-base` to squash without
```

Comment:
On other note, if it is required to be squashed into one commit, I thought the PR author no longer need to manually do that after bors is removed. I remember github provides ``squash and merge`` button https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/about-merge-methods-on-github?

Reply:
Right, a maintainer can do it on its own, but with the default behavior it produces commits which include the pull request name and its number while rebases do not. Therefore, it's better to adjust the behavior by switching `Default commit message` to `Pull request title` or other options rather than the default.